web-ui/eslint.config.js

```diff
@@ -31,6 +41,13 @@ export default [
       'react-hooks/exhaustive-deps': 'warn',
       ...vitestPlugin.configs.recommended.rules,
       'vitest/expect-expect': 'off',
+      '@typescript-eslint/no-unused-vars': 'off',
```

Comment:
Why do we want to turn these rules off?

Reply:
That was protoype code. It's been removed.